### DISCUSSION TITLE
A few newbie questions

### DISCUSSION THREAD

**Discussion by LukeAFullard** [quarto-dev/quarto-cli#1843] [Q&A] [answered]
Hello, I just came across quarto recently and have been enjoying making a few test html files.

I am trying to make a basic static website to display results of some water quality analyses and I had a few questions.
For context, I am looking at water quality in a number of **regions**, and will have separate individual pages for test **sites** within a region.

I want to make something approximating:

home page -
                   |
                   Region1  -- individual sites
                   |
                   Region2 -- individual sites
                   |

I hope that makes sense. I plan to have a side bar with links to the regions, but not to the sites.
The individual sites would be linked to via a clickable map or table from the region pages.

My question is this:
Should I generate all the individual sites html files (self-contained), then generate the regions pages, then the home page, or should I be generating everything using a single script at the same time?
Any other advice welcome.

Also, is there a Quarto discord?

Thanks

**Comment by mcanouil** [comment, marked as answer]
You can take a look at quarto's website, i.e., the "guide" menu in the navbar points to <https://quarto.org/docs/guide/> which is the home page of that category, but within it you have the left menu with many pages.
All the configuration and location of the pages is defined in the `_quarto.yml` file.
See <https://github.com/quarto-dev/quarto-web/blob/main/_quarto.yml>.

You can take a look at the website gallery: <https://quarto.org/docs/gallery/#websites>.
And/or to some website generated with Quarto and listed in <https://github.com/mcanouil/awesome-quarto#websites-formats>.
